deploy/crds/metrics_v1alpha1_appmetricsapp_cr.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: metrics.aerogear.com/v1alpha1
+kind: AppMetricsApp
```

Comment:
The kind looks a bit strange :)

Was it a mistake? Or if on purpose, can you explain what should resource do please?

Reply:
Ah, this would represent a client app...
Maybe AppMetricsClient is a better name?
Or AppMetricsSource?

Reply:
I'm basing it on the MSS consumption cr kind which is MobileSecurityServiceApp, aka <service>-app. It just happens that the name of this service has App in it already. Not sure I like it either. Thoughts on an alternative? In ups we use `variant` because that's what's created in the service instead of `app` which is what's created in the MSS service. There's no service here where something is created because of the bind. All we need to do is create the config map for use by MDC in mobile-services.json.

Reply:
Yeah, I'm not sure about the name of the installation CR either (AppMetricsService), happy to change that to if it helps.

@laurafitzgerald also note that in #2 I've changed the apiVersion to use .org instead of .com, to match with the other operators (I accidentally had it as .com when I created the project).

Reply:
sounds better than AppMetricsApp :) +1 on that

Reply:
+1 Going with `AppMetricsConfig`, will push the change soon.